core/openapi/schemas/reference.yaml

```diff
@@ -0,0 +1,7 @@
+type: object
+required:
+- "$ref"
+patternProperties:
```

Comment:
Personnally I replaced all patternProperties by additionalProperties. I think it is critical to keep support on swaggerhub. Moreover, 3.1 spec use another JSON Schema version: this is not a minor change !

Reply:
I copied these schema from the files @christophenoel referenced in this issues and I cleaned them up a bit.  The patternProperties key was in the full schema file.  I am not sure how I would replace that with additionalProperties since the patternProperties seems to be enforcing a rule that says that additional properties start with "x-".  So questions:
1. Do we need to enforce this "x-" rule for additional properties ?  If not, we can simple remove the patternProperties key.
2. If we want to keep the "x-" rules, then how would I replace patternProperties with additionalProperties and still enforce that additional properties must start with "x-"?
My vote would be to simply get rid of it since it seems that the "x-" rule is meant to allow the set of JSON schema keys to be arbitrarily extended with additional keys.  I doubt that anyone defining a process input or output would want to do that.